waltz-tools/src/test/java/com/wepay/waltz/tools/storage/StorageCliTest.java

```diff
@@ -502,12 +502,8 @@ public void testRecoverPartition() throws Exception {
             maxTransactionId = (Long) destinationClient.getMaxTransactionId(-1, partitionId).get();
             assertNotNull(maxTransactionId);
             assertEquals(9, maxTransactionId.longValue());
-
-            sourceClient.close();
```

Comment:
Why did you remove these two lines?

Reply:
Aha, good catch! Thought `helper.closeAll()` would also close `sourceClient`, but it is not true, will revert this change.